definitions.yaml

```diff
@@ -648,6 +651,57 @@ definitions:
                 type: string
                 description: Tenant ID of the Azure subscription
 
+  V4GetCredentialResponse:
+    type: object
+    description: Response model for getting details on a set of credentials
+    properties:
+      id:
+        type: string
+        description: Unique ID of the credentials
+      provider:
+        type: string
+        description: Either 'aws' or 'azure'
+      aws:
+        type: object
+        description: Credentials specific to an AWS account
+        properties:
+          roles:
+            type: object
+            description: IAM roles to assume by certain entities
+            properties:
+              admin:
+                type: string
+                description: ARN of the IAM role Giant Swarm support staff will use
+              awsoperator:
+                type: string
+                description: ARN of the IAM role assumed by the software operating clusters
```

Comment:
`ARN of the IAM role assumed by the software operating the clusters` reads a bit easier for me

Reply:
changed